data/sheragi/archaeology missions.txt

```diff
@@ -1268,12 +1293,11 @@ mission "Sheragi Archaeology: The Emerald Sword 3"
 
 	on enter "Zubenelhakrabi"
 		conversation
-			`The trip to Zubenelhakrabi is completely normal and calm. Excessively so for everyone's overflowing expectations though, the trip's uneventfulness only managing to keep everyone on edge. When you arrive, a simple scan reveals the system's star is unremarkable, nearing the end of its main sequence phase. Among the planets orbiting it, one is in the habitable zone and shows apparent signs of life, covered in dark green spots under the clouds.`
+			`The trip to Zubenelhakrabi is completely normal and calm. The trip's uneventfulness only managing to keep everyone on edge. When you arrive, a simple scan reveals the system's star is unremarkable, nearing the end of its main sequence phase. Among the planets orbiting it, one is in the habitable zone and shows apparent signs of life, covered in dark green spots under the clouds.`
```

Comment:
Even after some changes made here, the first part of this second sentence still sounds weird to me. Decided to just ax it.

Reply:
I would make those first two sentences into a single one then, separated by a comma. It's a good solution though.